docker/structs.go

```diff
@@ -0,0 +1,8 @@
+package docker
+
+import "github.com/docker/engine-api/types"
+
+// VolumeRequest is taken from
+// https://github.com/docker/engine-api/types/types.go
+type VolumeRequest types.VolumeCreateRequest
```

Comment:
ah, may be I spoke too soon :) 

Here is what docker is proposing to use: https://github.com/docker/go-plugins-helpers/tree/master/volume